multimod/internal/sync/sync.go

```diff
@@ -120,10 +144,20 @@ func (s sync) updateAllGoModFiles() error {
 		modFilePaths = append(modFilePaths, filePath)
 	}
 
+	ver := s.OtherModuleSet.Version
+	if s.OtherModuleVersionTag != "" {
+		version, err := s.parseVersionInfo(string(s.OtherModuleSet.Modules[0]), s.OtherModuleVersionTag)
+		if err != nil {
+			return err
+		}
+		ver = version
+	}
+	log.Printf("Version: %s\n", ver)
```

Comment:
Stray logging? Should we remove it?

Reply:
This was left here intentionally to make it obvious to the person running the command what version was used in the end